Jenkinsfile

```diff
@@ -1,32 +1,50 @@
-pipeline {
+@Library('releng-pipeline@main') _
 
+pipeline {
   agent {
-    label 'docker-build'
+    kubernetes {
+      yaml loadOverridableResource(
+        libraryResource: 'org/eclipsefdn/container/agent.yml'
+      )
+    }
   }
 
-  options { 
+  options {
     buildDiscarder(logRotator(numToKeepStr: '5'))
-    disableConcurrentBuilds()
```

Comment:
Any good reason why `disableConcurrentBuilds()` was removed?

Reply:
This is a mistake, I copied something similar between all docker projects. I have added it again, and will applied same modification to other projects.